.github/workflows/test.yml

```diff
@@ -31,6 +31,11 @@ jobs:
         run: bin/test.sh
```

Comment:
please split test.sh into unit-test.sh and bdd-test.sh so its easier to find test results etc.

Reply:
Done, I just invoked the ahoy steps directly.